src/js/server/handler.js

```diff
@@ -0,0 +1,46 @@
+// An extension of `gpii.express.handler` that adds JSON Schema headers to the outgoing response.  This is intended to
```

Comment:
These kinds of terse filenames are a bit of a hazard when searching in a crowded IDE - could we have this as schemaHandler.js and the one below as schemaMiddleware.js ?